Mathlib/Data/Bitvec/Lemmas.lean

```diff
@@ -21,8 +21,17 @@ open Nat
 
 variable {w v : Nat}
 
-theorem toNat_inj {x y : BitVec w} : x.toNat = y.toNat ↔ x = y :=
-  ⟨(match x, y, · with | ⟨_, _⟩,⟨_, _⟩, rfl => rfl), (· ▸ rfl)⟩
```

Comment:
You could have just kept the first half of this proof :)

Reply:
I gave it another shot, but I am afraid it is not yet minimal. Advice is much appreciated, otherwise we will try to reduce it further on Monday.